kernel/cluster.c

```diff
@@ -43,16 +42,24 @@ static void cluster_core_init()
     eu_bar_setup(eu_bar_addr(0), (1<<ARCHI_CLUSTER_NB_PE) - 1);
 }
 
+
```

Comment:
There are a few whitespace changes in this PR, some adding unneeded spaces. Can you remove these?

Reply:
Of course

Reply:
I hope I removed all the whitespaces!